src/types/kongponents-plugin.ts

```diff
@@ -0,0 +1,12 @@
+import { v4 as uuidv4 } from 'uuid'
+
+export const kongponentsRandomIdFunction = 'kongponents-id' as const
```

Comment:
nit: can we rename this a bit:
```suggestion
export const kongponentsGetUniqueId = 'kongponents-get-unique-id' as const
```